modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Dockerfile.standalone

```diff
@@ -0,0 +1,6 @@
+﻿FROM mcr.microsoft.com/dotnet/core/runtime:3.1
+
+COPY ./publish /publish
```

Comment:
Looks like this `Dockerfile.standalone` relies on the other `Dockerfile` to perform the build. Can this please be changed to to also contain the build step itself so that it is self-sufficient ?

Reply:
no, it is just for dockerbuild.bat, no build steps needed here

Reply:
ah, ok. i see the build and publish steps there.